libs/core/src/lib/micro-process-flow/components/micro-process-flow-icon/micro-process-flow-icon.component.ts

```diff
@@ -0,0 +1,33 @@
+import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+import { IconFont } from '@fundamental-ngx/core/icon';
+
+@Component({
+    selector: 'fd-micro-process-flow-icon',
+    templateUrl: './micro-process-flow-icon.component.html',
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
We don't need this rule `encapsulation: ViewEncapsulation.None,`  here because we don't import any global styles. 
And probably  no need this rule too, because we already have it on parent component.
` changeDetection: ChangeDetectionStrategy.OnPush,`

Reply:
I've checked that almost all components in core have OnPush strategy, doesn't matter if it's a child item of some another component.
I've checked with Default strategy, and amount of checks increases, so I wouldn't advise doing that.

Reply:
Ok, What about ViewEncapsulation.None? Why do we use it here?

Reply:
If we don't encapsulate the root component, why should we do it for the child ones?